updating/updating-cluster-between-minor.adoc

```diff
@@ -0,0 +1,22 @@
+[id="updating-cluster-between-minor"]
+= Updating a cluster between minor versions
+include::modules/common-attributes.adoc[]
+:context: updating-cluster-between-minor
+
+toc::[]
+
+You can update, or upgrade, an {product-title} cluster between minor versions.
+
+[IMPORTANT]
+====
+You must update your cluster to a new minor version by using the web console.
+====
```

Comment:
I've gotten conflicting messages about whether or not it's required to use `--force` from the CLI. After update between 4.1 and 4.2 is working as intended, I'm happy to revisit.

There's an ongoing discussion about how the update from 4.1.z to 4.2 is supposed to work on the sbr-shift-list, and I've asked Eric to let me know when there's a plan to move forward.

Reply:
We don't want customers forcing upgrades, we want them using the graphs we provide. 
That said we need to document to customers how to use the force option, and clearly explain that its not a recommended decision to do this.

Reply:
Customers can update from cli or web console, the web console (or our Cincinnati graph are the ways you know what acceptable upgrades are).

Reply:
I'm happy to add some conceptual material and warnings about the existence of the --force option and emphasize the need to use one of the supported options in the graph. We're only adding steps that we actually want users to take. (If I add steps about how to use the --force option, you know people are going to follow them.)

Reply:
> Can i understand this "user can only upgrade to a minor version through web console"? why not cli supported in minor version upgrade?
> and for a 4.1.z version, how user can upgrade the cluster to 4.2? i ask this because i did not see any available update for both 4.1.18 and 4.1.20 when set channel to stable-4.2?

@jiajliu, Trevor and I talked about this for awhile. It's because you need to change channel from the `stable-4.1` to a 4.2 channel to upgrade, and there's not a good way to do that in the CLI. Instead of changing channel in the UI and then swapping to the CLI to finish, we're just staying in the UI.